pkg/app/app_test.go

```diff
@@ -281,3 +282,33 @@ func Test_initConfiguration_PREVIEW_FEATURES_ENABLED(t *testing.T) {
 	actual = config.GetBool(configuration.PREVIEW_FEATURES_ENABLED)
 	assert.True(t, actual)
 }
+
+func Test_initConfiguration_DEFAULT_TEMP_DIRECTORY(t *testing.T) {
```

Comment:
Suggestion: add a case where the tmp path is specified in the config, this should return the configured rather then the default value.